src/app/modules/OpenshiftStreams/dialogs/CreateInstance/CreateInstance.tsx

```diff
@@ -51,52 +52,26 @@ const CreateInstanceWithSizes: FunctionComponent<
     [hideModal, onCreate, createInstance]
   );
 
-  const getAvailableProvidersAndDefaults =
-    useCallback(async (): Promise<CreateKafkaInitializationData> => {
-      const data = await fetchAvailableProvidersAndDefault();
-      capabilitiesRef.current = data;
-      return data;
-    }, [fetchAvailableProvidersAndDefault]);
-
-  const kafkaSizes = useCallback<CreateKafkaInstancePropsWithSizes["getSizes"]>(
-    (provider, region) => {
-      return new Promise((resolve, reject) => {
-        const capabilities = capabilitiesRef.current;
-        if (!capabilities) {
-          reject("Unexpected error, missing provider data");
-        } else {
-          const providerInfo = capabilities.availableProviders.find(
-            (p) => p.id === provider
-          );
-          const regionInfo = providerInfo?.regions.find((r) => r.id === region);
-          const availableSizes =
-            regionInfo?.capacity.flatMap((c) =>
-              c.available_sizes.map((s) => `${c.instance_type}.${s}`)
-            ) || [];
-          resolve(getKafkaSizes(provider, region, availableSizes));
-        }
-      });
-    },
-    [getKafkaSizes]
-  );
-
   return (
-    <CreateKafkaInstanceWithSizes
+    <CreateKafkaInstance
+      checkDeveloperAvailability={checkDeveloperAvailability}
+      checkStandardQuota={checkStandardQuota}
+      fetchProvidersWithRegions={fetchProvidersWithRegions}
+      getStandardSizes={getStandardSizes}
+      getTrialSizes={getTrialSizes}
       isModalOpen={true}
-      onClickQuickStart={onClickQuickStart}
       onCancel={hideModal}
-      getAvailableProvidersAndDefaults={getAvailableProvidersAndDefaults}
-      onCreate={handleCreate}
       onClickContactUs={onClickKafkaOverview}
-      onClickLearnMoreAboutRegions={onClickKafkaOverview}
+      onClickKafkaOverview={onClickKafkaOverview}
+      onClickQuickStart={onClickQuickStart}
+      onCreate={handleCreate}
       onLearnHowToAddStreamingUnits={onClickKafkaOverview}
       onLearnMoreAboutSizes={onClickKafkaOverview}
-      onClickKafkaOverview={onClickKafkaOverview}
-      getSizes={kafkaSizes}
+      subscriptionOptionsHref={"TODO"}
```

Comment:
@jgiardino do we have the link for this already?

Reply:
Without seeing this in context of the UI, I _think_ this is a link to the Kafka overview page, where we include links to the various options for getting a subscription.

Reply:
👍